content/language-agnostic/encyclopedia/what-is-an-api.md

```diff
@@ -0,0 +1,28 @@
+---
+Title: "What is an API?"
+Subjects:
+  - "Code Foundations"
+  - "Computer Science"
+  - "Web Development"
+  - "Web Design"
+Tags:
+  - "APIs" 
+  - "Developer Tools"
+  - "Dependency"
+  - "Interface"
+Catalog Content:  
+  - "https://www.codecademy.com/learn/paths/computer-science"
+  - "https://www.codecademy.com/learn/paths/web-development"
+  - "https://www.codecademy.com/learn/ibm-watson"
```

Comment:
Let's take out the `ibm-watson` and `learn-alexa` courses just because we haven't updated them in a while.